### ISSUE TITLE
[DX][UX] Link this repo from the web-pages

### ISSUE THREAD

**Issue by webknjaz** [scikit-hep/scikit-hep.github.io#69]
I've navigated to the web-page from https://github.com/pypa/gh-action-pypi-publish/issues/15#issuecomment-663999259 and then saw a typo there. I wanted to send a patch (https://github.com/scikit-hep/scikit-hep.github.io/pull/70) but It took me about 10-15 min just to find this repo.

Please explore how it could be made better discoverable. Maybe add a corner-banner saying "edit this page on GitHub" or something like that...

**Comment by eduardo-rodrigues** [comment]
The link is obvious, though, so I cannot understand how it took you 10-15 minutes (BTW a search via Google would also get there in seconds): indeed, if you went to https://scikit-hep.org/developer/gha_wheels via the page you give above then the link "Scikit-HEP on GitHub" is just at the top. Scroll a bit down and you immediately get to scikit-hep.github.io.

**Comment by webknjaz** [comment]
> Scroll a bit down and you immediately get to scikit-hep.github.io.

Well, this "immediately" part depends on when it was last updated. It also needs people to know upfront that the site is hosted on GH Pages and that GH Pages have a special naming pattern. There's nothing obvious about it.

**Comment by eduardo-rodrigues** [comment]
Then ask just below the comment where you saw a mention of it, which would be to @henryiii at https://github.com/pypa/gh-action-pypi-publish/issues/15#issuecomment-663999259? Personally I find communication rather effective ...

**Comment by webknjaz** [comment]
Hey, I'm not trying to solve my problem anymore. I just suggested making it easier for a random person on the internet. I considered asking but was hoping that sending a PR is faster (it usually is!). I didn't want to comment on that issue because it'd be unrelated and I didn't want to spam other discussions participants with this + the feedback loop with asking is longer. For somebody wanting to send a one-char typo drive-by PR, it's way too much complexity for such a simple action. In fact, I was considering giving up as it was starting to be not worthy of my time...

**Comment by eduardo-rodrigues** [comment]
Glad you sent the fix for the typo got unnoticed. Thanks.

**Comment by henryiii** [comment]
Hi @webknjaz,

The link "Scikit-HEP on GitHub" got moved when we developed an issue with our stylesheet (still to be investigated in #65, though I think it could be the style we base ours on got updated to Jekyll 4.0 and GitHub pages is still stuck on 3.x). It was on the far right, a fairly common place for such a link. Not it's in a rather odd place mid left.
As you've pointed out, it's not simple to get to the correct page to edit from there - a common item for similar pages is an "edit this page" (for example, in most readthedocs pages). I don't think that would be a bad idea, honestly, to jump more quickly to at least the repo itself and suggest a change. I'm a huge fan of quick links to source from pages, and pages from source. Even for experienced users, it's much easier to navigate around, and it's crucial for inexperienced users. I'm usually not a fan of being thrown directly into the edit page mode, but for a page that is not documentation for source code, even that might not be a bad idea.
